src/pymor/analyticalproblems/expressions.py

```diff
@@ -393,6 +443,9 @@ def __init__(self, operand):
     def numpy_expr(self):
         return f'(- {self.operand.numpy_expr()})'
 
+    def fenics_expr(self, params):
+        return np.vectorize(lambda x : -1 * x)(self.operand.fenics_expr(params))
```

Comment:
Is multiplication by -1 really necessary, or does

```suggestion
        return np.vectorize(lambda x : -x)(self.operand.fenics_expr(params))
```

work as well?